testdata/catalogs/test-catalog/catalog.json

```diff
@@ -0,0 +1,33 @@
+{
+    "schema": "olm.package",
+    "name": "prometheus",
+    "defaultChannel": "beta"
+}
+{
+    "schema": "olm.channel",
+    "name": "beta",
+    "package": "prometheus",
+    "entries": [
+        {
+            "name": "prometheusoperator.0.47.0"
+        }
+    ]
+}
+{
+    "schema": "olm.bundle",
+    "name": "prometheusoperator.0.47.0",
+    "package": "prometheus",
+    "image": "localhost/testdata/bundles/registry-v1/prometheus-operator:v0.47.0",
+    "properties": [
+        {
+            "type": "olm.package",
+            "value": {
+                "packageName": "prometheus",
+                "version": "0.47.0"
+            }
+        }
+    ],
+    "relatedImages": [
+        {}
+    ]
```

Comment:
Nit: remove `"relatedImages`" entirely?

Reply:
It looks like spec.relatedImages field is not optional: 

```
condition "Unpacked" is False/UnpackFailed; message: "create bundle metadata objects: applying bundle metadata \"test-catalog-prometheusoperator.0.47.0\": BundleMetadata.catalogd.operatorframework.io \"test-catalog-prometheusoperator.0.47.0\" is invalid: spec.relatedImages: Invalid value: \"null\": spec.relatedImages in body must be of type array: \"null\""
```

Admittedly I'd have preferred to keep it at `"relatedImages": []` but even that is read as null, and had to do some json jujitsu to come up with `[{}]`

Reply:
^ Shouldn't be an issue after https://github.com/operator-framework/catalogd/issues/38 is done

Reply:
We'll have to create a follow up issue to fix this.